include/dynamatic/Dialect/Handshake/HandshakeTypes.td

```diff
@@ -367,6 +374,16 @@ class AllExtraSignalsMatchWithVariadicExcept<string except, string variadic, lis
   >.result # ", \"" # except # "\")">
 >;
 
+/// Constraint to ensure that HandshakeType has no extra signals.
+class LacksExtraSignals<string operand> : PredOpTrait<
```

Comment:
I think this is equivalent to `IsSimpleHandshake`. I prefer the name of `LacksExtraSignals` though. Can we rename it? (cc: @murphe67)